codecov.yml

```diff
@@ -0,0 +1,10 @@
+coverage:
+  range: 85..100
+  status:
+    project:
+      default:
+        threshold: 1%
```

Comment:
I think we want a comment telling this is due to flaky tests and we should make it smaller once they are fixed.